simple_network_sim/network_of_populations.py

```diff
@@ -83,10 +100,26 @@ def basicSimulationInternalAgeStructure(network, timeHorizon):
 
 # CurrentlyInUse
 def totalIndividuals(nodeState):
+    """Calculate the total population size over all ages and regions.
```

Comment:
This function takes a node (region) and counts individuals from every age and state.